src/reanimated2/hook/useAnimatedSensor.ts

```diff
@@ -52,18 +124,42 @@ export function useAnimatedSensor(
     isAvailable: false,
     config: {
       interval: 0,
+      adjustToInterfaceOrientation: false,
+      iosReferenceFrame: IOSReferenceFrame.XArbitraryCorrectedZVertical,
     },
+    interfaceOrientation: makeMutable<InterfaceOrientation>(0),
   });
 
   useEffect(() => {
-    ref.current.config = { interval: 'auto', ...userConfig };
+    ref.current.config = {
+      interval: 'auto',
+      adjustToInterfaceOrientation: false,
```

Comment:
Is there any reason that prevents us from enabling this by default?
```suggestion
      adjustToInterfaceOrientation: true,
```

Reply:
Yes, though I'm not sure myself. Whether we should adjust or not and how to do it depends on the use case. On the other hand this should work for most cases, so I'll change it.

Reply:
Yeah, let's make it beginner-friendly out-of-the-box.